pyserini/search/_base.py

```diff
@@ -100,6 +100,10 @@ def get_topics(collection_name):
         topics = JTopicReader.getTopicsWithStringIds(JTopics.CAR17V15_BENCHMARK_Y1_TEST)
     elif collection_name == 'car17v2.0_benchmarkY1test':
         topics = JTopicReader.getTopicsWithStringIds(JTopics.CAR17V20_BENCHMARK_Y1_TEST)
+    elif collection_name == 'dl19_doc':
+        topics = JTopicReader.getTopicsWithStringIds(JTopics.TREC2019_DL_DOC)
+    elif collection_name == 'dl19_passage':
+        topics = JTopicReader.getTopicsWithStringIds(JTopics.TREC2019_DL_PASSAGE)
```

Comment:
this means we need to do a artifact deploy on the Anserini end for this to work right?

Reply:
yeah, we already have these on the anserini end